.gitpod.yml

```diff
@@ -7,15 +7,14 @@ tasks:
     - init: >
         bundle install --without production &&
         cp config/database.yml.gitpod config/database.yml &&
-        mysql -e "SET @@global.sql_mode=(SELECT REPLACE(@@global.sql_mode, 'ONLY_FULL_GROUP_BY', ''));" &&
-        mysql -e "CREATE DATABASE plots;" &&
         cp db/schema.rb.example db/schema.rb &&
         rake db:setup &&
         yarn install
 
       command: >
         passenger start
-
+        
+    - command: redis-server
```

Comment:
Maybe instead of creating a new `command` task you can just include the `redis-server` below the `passenger-start` since the `>` supports multiline bash.